### ISSUE TITLE
Propose a strategy for hold/recalls for multi-volume works

### ISSUE THREAD

**Issue by cbeer** [sul-dlss/sul-requests#1453]

**Comment by jcoyne** [comment]
Should we consider this a bug in Folio and file it upstream?

**Comment by jcoyne** [comment]
@cbeer I agree that we need to fix it regardless, but it's still interesting to me whether we think of this as a bug in Folio.

**Comment by jcoyne** [comment]
Could we prevent title level requests from being made if it's a multi-volume work?  Then we leave it up to the patron to select the correct item.
